src/app/notification-view/notification-view.component.ts

```diff
@@ -0,0 +1,18 @@
+import { Component, OnInit } from '@angular/core';
+import { ActivatedRoute } from '@angular/router';
+
+@Component({
+  template: `{{message}}`,
+})
+export class NotificationViewComponent implements OnInit {
+  message: string;
+  constructor(
+    private route: ActivatedRoute
+  ) {}
+
+  ngOnInit() {
+    this.route.queryParams.subscribe(params => {
+        this.message = params['message'];
```

Comment:
Indentation is off, and after switching to use a route parameter you'll need to grab the notification info from the database.